modules/labs-react/layout/stories/Stack.stories.mdx

```diff
@@ -0,0 +1,75 @@
+import {Meta} from '@storybook/addon-docs/blocks';
+
+import {Stack} from '@workday/canvas-kit-labs-react/layout';
+// examples
+import {NestedStacks} from './examples/NestedStacks';
+import {StackCard} from './examples/StackCard';
+import {HStackCards} from './examples/HStackCards';
+import {VStackCards} from './examples/VStackCards';
+
+<Meta title="Labs/Layout/Stack" component={Stack} />
+
+# Stack, HStack, and VStack
+
+[![LABS: Beta](https://img.shields.io/badge/LABS-beta-orange)](https://github.com/Workday/canvas-kit/tree/master/modules/labs-react/README.md)
+This component is work in progress and currently in pre-release.
+
+`Stack` is a higher-level layout component that provides an ergonomic API for building one-dimensional layouts with consistent spacing between elements. It's built on top of `Flex` and has access to all `Flex` and `Box` props. `Stack` can align items in either a horizontal or vertical orientation. `HStack` and `VStack` will only align items in a horizontal or vertical orientation, respectively.
+
+## Stack
+
+The most important distinction to make with `Stack` is that it is _not a grid._ Grid components are able to align items along two dimensions (rows and columns), and `Stack` is only built to support one dimension (rows _or_ columns). That said, you can nest horizontal and vertical `Stack` components to create grid-like layouts.
+
+The direction of the child elements is set with the `flexDirection` or  `direction` prop (a shorthand alias). These props support four directions: `column`, `column-reverse`, `row`, and `row-reverse`.
```

Comment:
I know the `flexDirection` prop is rooted in the `flex-direction` property of flexbox so perhaps we shouldn't change this, but I've always found `column` and `row` to be confusing. I see `flex-direction: column` and think "columns," expecting the children to be arranged left-to-right like columns. `vertical` and `horizontal` seem clearer to me.

A [Google search](https://www.google.com/search?q=flex-direction+confusing&oq=flex-direction+confusing) reveals some nuance and history behind this decision so maybe we shouldn't mess with this.

Reply:
Yeah, my only other thought is that the prop could be called `stackDirection`, but I think `flexDirection` is probably better because we have an existing mental model to build upon with Flexbox.